src/features.json

```diff
@@ -67,6 +67,20 @@
     "THE_TRADE_DESK": true,
     "INTERCOM": true
   },
+  "regulation": [
```

Comment:
Just highlighting, not relevant to this pr. We don't have user delete tests for iterable & custify destinations. Better to add them sometime soon.

Reply:
Should we call it `regulations`?